ASFAchievementManager/AchievementHandler.cs

```diff
@@ -293,16 +289,30 @@ public sealed class AchievementHandler : ClientMsgHandler {
 				}
 			};
 			request.Body.stats.AddRange(statsToSet);
-			Responses.TryRemove(appId, out StoredResponse? Dummy);
 			Client.Send(request);
-			SpinWait.SpinUntil(() => Responses.ContainsKey(appId), TimeSpan.FromSeconds(ASF.GlobalConfig?.ConnectionTimeout ?? GlobalConfig.DefaultConnectionTimeout));
-			if (!Responses.TryGetValue(appId, out StoredResponse? storeResponse)) {
-				responses.Add(Strings.WarningFailed);
-			}
-			responses.Add((storeResponse?.Success ?? false) ? Strings.Success : Strings.WarningFailed);
+
+			var setResponse = await new AsyncJob<SetAchievementsCallback>(Client, request.SourceJobID).ToLongRunningTask().ConfigureAwait(false);
```

Comment:
This project still does not use the `var` keyword

Reply:
Corrected.